lib/et/src/et_collector.erl

```diff
@@ -438,6 +503,10 @@ report_event(CollectorPid, DetailLevel, From, To, Label, Contents)
 %% Key = record(event_ts) | record(trace_ts)
 %%----------------------------------------------------------------------
 
+-spec make_key(TH, Stuff) -> Key when
+      TH :: table_handle() | trace_ts | event_ts,
+      Stuff :: event() | Key,
+      Key :: event_ts() | trace_ts().
```

Comment:
The docs calls TH Type which makes more sense!

Reply:
Or maybe Handle could be good.

Reply:
I'll update to `Handle`